identity-service/src/routes/trackListens.js

```diff
@@ -174,6 +174,9 @@ const getTrendingTracks = async (
       let oneYearBefore = new Date(currentHour.getTime() - oneYearInMs)
       dbQuery.where.hour = { [models.Sequelize.Op.gte]: oneYearBefore }
       break
+    case 'millennium':
+      dbQuery.where.hour = { [models.Sequelize.Op.gte]: new Date(0) }
```

Comment:
nit: could just do `new Date()`

Reply:
new Date() does today not beginnign of time :)